src/lib/constants.ts

```diff
@@ -30,7 +29,13 @@ export const enum SupportedGame {
 export function getGameTitle(game: SupportedGame) {
   switch (game) {
     case SupportedGame.Jak1:
-      return TranslatedStrings.jak1_gameName;
+      return constant.jak1_gameName;
+    case SupportedGame.Jak2:
+      return constant.jak2_gameName;
+    case SupportedGame.Jak3:
+      return constant.jak3_gameName;
+    case SupportedGame.JakX:
+      return constant.jakX_gameName;
```

Comment:
This _might_ break some functionality. Leave it for now, but make sure you test all app functionality before I take the time to test too.

Reply:
I testet it and it should work for all, because the file doesnt change at all.